app/src/main/res/layout/activity_main.xml

```diff
@@ -5,7 +5,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <androidx.emoji.widget.EmojiTextView
+    <androidx.emoji2.widget.EmojiTextView
```

Comment:
🔍 Since it uses AppCompat, `TextView` supports rendering emojis so there's no longer need to use `EmojiTextView` 😄

Reply:
Yeah as I said in the description, for the sample code I decided to use `emoji2` just to showcase the migration/usage of `emoji2` library from a library-user's perspective. If you think there is no value to it I can just use AppCompat

Reply:
Oh, and I read the description but didn't pay enough attention it, apologies 🙈 
With that reasoning I think it's ok, for the final version bump to `2.x` I think it'd be easier on users to not have this to avoid confusion, what do you think?

Reply:
I can agree on that. At the end of the day `emoji` documentation is thorough enough so if someone wants to use it they can refer to official documentation.

I'll proceed with AppCompat changes in this PR!

Reply:
Resolved in [048427a7](https://github.com/withplum/EmojiBottomSheetDialog/pull/19/commits/048427a72d907990743755c13028df75803b666b)